src/components/Editor/Properties/PropertyText.vue

```diff
@@ -96,6 +97,23 @@ export default {
 }
 </script>
 <style lang="scss" scoped>
+.property-text {
+	position: relative;
+}
+
+.property-text__icon {
+	position: absolute;
+	top: calc(var(--default-grid-baseline) * 1);
+}
+
+.property-text__input {
+	padding-left: calc(var(--default-grid-baseline) * 11);
+}
+
+.property-text__readonly-value {
+	white-space: pre-wrap;
```

Comment:
```suggestion
	white-space: pre-wrap;
	word-break: break-all;
```

This will also cause very long words/links to be wrapped correctly.

Reply:
Done